### PR TITLE
Feature/add poster

### PR DESCRIPTION
**Pull request by HumnaAleem**

### CODE REVIEW

File: consumer/src/pages/Challenge/index.tsx [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+
+function Challenge(props: any) {
+	return <div>
+		<div className="poster-div" style={{
+		}}>
+			<h3 className="independence-heading">Happy Independence Day</h3>
+			<div className="content-div">
+				<div className="quotes-div">
+					<div className="quotes">
+						Lorem ipsum dolor sit amet, consectetur adipiscing elit. Aenean a ullamcorper risus, at euismod sem. Fusce tincidunt quam purus, in sagittis nisi tristique ut. Duis eget elementum neque. Vivamus posuere bibendum quam in interdum. Cras ac quam quam. Morbi eu enim vestibulum, ultrices metus et, porttitor enim. Sed volutpat mauris vitae ex lobortis, id lobortis felis dictum. Praesent blandit nisi cursus nunc venenatis, sit amet porta mauris semper. Integer pretium dui in rutrum finibus. Quisque vel mi et sem tincidunt mattis. Mauris dapibus, enim sit amet tincidunt volutpat, orci dolor laoreet est, et sagittis ipsum lectus eu nibh. Morbi efficitur est eget elit congue, eu dictum justo semper. Quisque vitae elementum risus. Ut quis diam venenatis, eleifend purus in, pulvinar mauris. Sed vitae arcu nibh. Nulla in maximus metus.</div>
```
Comment (raoaliahmad) [inline consumer/src/pages/Challenge/index.tsx:14]: remove this we don't have any text, just show the poster

File: consumer/src/components/Header/index.tsx [hunk 2]
```diff
@@ -16,6 +16,7 @@ const Header = ({ path, history }: Props) => {
 			(path !== "/" && path !== "/about-us") && <img src={BackIcon} style={{ width: "20px", marginRight: "10px" }} onClick={() => history.goBack()} alt="back-arrow" />
 		}
 		<Link to="" className="logo" />
+		<Link to="/challenge" className="challenge-btn">Challenge</Link>
```
Comment (raoaliahmad) [inline consumer/src/components/Header/index.tsx:19]: this is fine for now, but I think for future we would need to add a top down burger menu

Comment (HumnaAleem) [reply consumer/src/components/Header/index.tsx:19]: ok

File: consumer/src/pages/Challenge/index.tsx (hunk 1 above)
Comment (HumnaAleem) [reply consumer/src/pages/Challenge/index.tsx:14]: we can add quotes or something else . without it, it seems empty.

Comment (raoaliahmad) [reply consumer/src/pages/Challenge/index.tsx:14]: It's up to you If you want to add quotes, but they haven't given us any so you would have to find one yourself. Also these quotes/text would look better on top or below the poster not on the side

**Pull request merged**